crates/proto/Cargo.toml

```diff
@@ -72,19 +87,25 @@ backtrace = { workspace = true, optional = true }
 bitflags = { workspace = true, optional = true }
 bytes = { workspace = true, optional = true }
 cfg-if.workspace = true
-data-encoding.workspace = true
+const-random = { version = "0.1.15" }
```

Comment:
Don't revert to using specific versions, all dependency versions should be tracked in the workspace `Cargo.toml`.